app.js

```diff
@@ -35,23 +35,12 @@ app.use((err, req, res, next) => {
       error: err
     }
   });
-  next(err);
```

Comment:
Why are we taking out this error handling?

Reply:
I removed it because presently it is not doing anything in the development. It is for production. I did not want to comment it out.